pkg/auth/credential_test.go

```diff
@@ -0,0 +1,82 @@
+package auth
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/notaryproject/notation/pkg/config"
+)
+
+const (
+	errMsg     = "error message"
+	validStore = "pass"
+)
+
+func TestLoadConfig_loadNotationConfigFailed(t *testing.T) {
```

Comment:
Should we use an upper-case name `TestLoadConfig_LoadNotationConfigFailed` instead of `TestLoadConfig_loadNotationConfigFailed`?

Reply:
Right, I'll get it updated.